README.md

```diff
@@ -591,10 +591,10 @@ you proceed to [step 4](#verify-itts-client-version).
 
 #### 9.1. Event log and measurements
 
-One of the key components remote attestation is based on is the runtime measurement values (stored in RTMRs
-registers for each TD by the TDX module). These RTMR values are computed from the digests of the entries of
-the TD boot event log. During the system's boot, each event's digest (SHA384) is extended (appended) to the
-current value of the associated RTMR. The resulting value's digest becomes the new value of the RTMR.
+One of the key components remote attestation is the runtime measurement values (stored in RTMRs
+registers for each TD by the TDX module). During the system boot, each component of the boot process (binary or conf)
+is measured into a digest. This digest value is extended (appended) to the RTMR's current value. The digest of the result value
```

Comment:
Same thing with appended/extended. 

Also, I'm not sure what is meant by "measured into a digest". I think what you're meaning here is that each component of the boot process becomes input for the digest/hash function?

Reply:
digest is the value from the hash function that is applied to data that needs to be measured